src/components/erc721/common/inputSearch.tsx

```diff
@@ -35,9 +42,15 @@ const SearchInput = styled.input`
     }
 `;
 
-export class Search extends Component {
+export class Search extends React.Component<Props> {
     public render = () => {
-        const { ...restProps } = this.props;
-        return <SearchInput {...restProps} placeholder={'Search Cryptocards'} />;
+        const { placeHolder, onChange, ...restProps } = this.props;
+        return (
+            <SearchInput
+                {...restProps}
+                placeholder={placeHolder ? placeHolder : 'Search Cryptocards'}
```

Comment:
I know it was used before, but I don't think we should use this default. Let's just make the placeholder mandatory. (Also, it's `placeholder`, not `placeHolder`).

Reply:
Done